File: Bab 8/Exercise/4. Print Character.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String[] string = new String[10];
		int banyak, baris, kolom;

		Scanner sc = new Scanner(System.in);

		banyak = 0;
		baris = 0;
		kolom = 0;

		try{

			banyak = sc.nextInt();

		}catch(Exception e){

		}

		for (baris = 0; baris < banyak; baris++) {

			try{

				string[baris] = sc.next();

			}catch(Exception e){

			}
		}

		for (baris = 0; baris < banyak; baris++)
		{

			if ((string[baris].charAt(0) == 'A') || (string[baris].charAt(0) == 'a'))
			{
				
				for (kolom = string[baris].length() - 1; kolom >= 0; kolom--)
				{
					System.out.print(string[baris].charAt(kolom));
				}

				System.out.println();
			
			}

			else if ((string[baris].charAt(0) == 'I') || (string[baris].charAt(0) == 'i'))
			{
				int length = 0;
				int start = 0;
				int middle = 0;

				length = string[baris].length();

				if (length % 2 == 0)
				{
					start = (length / 2) - 1;
					middle = length / 2;
				}

				else{
					start = length / 2;
					middle = (length / 2) + 1;
				}

				for (kolom = start; kolom >= 0; kolom--)
				{
					System.out.print(string[baris].charAt(kolom));
				}

				for (kolom = middle; kolom < length; kolom++)
				{
					System.out.print(string[baris].charAt(kolom));
				}

				System.out.println();
			}

			else if ((string[baris].charAt(0) == 'U') || (string[baris].charAt(0) == 'u'))
			{
				for (kolom = string[baris].length() - 1; kolom >= 0; kolom--)
				{
					System.out.print(string[baris].charAt(kolom));
				}

				System.out.println();
			}

			else{
				
				System.out.println(string[baris]);
			}	
		}
	}
}

File: Bab 8/Exercise/4. Print Character.c
#include <stdio.h>
#include <string.h>

int main()
{

	char string[99][99];
	int banyak, baris, kolom;

	scanf("%d", &banyak);

	for (baris = 0; baris < banyak; baris++)
	{
		scanf(" %s", &string[baris]);
	}

	for (baris = 0; baris < banyak; baris++)
	{

		if ((string[baris][0] == 'A') || (string[baris][0] == 'a'))
		{
			
			for (kolom = strlen(string[baris]) - 1; kolom >= 0; kolom--)
			{
				printf("%c", string[baris][kolom]);
			}

			printf("\n");
		
		}

		else if ((string[baris][0] == 'I') || (string[baris][0] == 'i'))
		{
			int length = 0;
			int start = 0;
			int middle = 0;

			length = strlen(string[baris]);

			if (length % 2 == 0)
			{
				start = (length / 2) - 1;
				middle = length / 2;
			}

			else{
				start = length / 2;
				middle = (length / 2) + 1;
			}

			for (kolom = start; kolom >= 0; kolom--)
			{
				printf("%c", string[baris][kolom]);
			}

			for (kolom = middle; kolom < length; kolom++)
			{
				printf("%c", string[baris][kolom]);
			}

			printf("\n");
		}

		else if ((string[baris][0] == 'U') || (string[baris][0] == 'u'))
		{
			for (kolom = strlen(string[baris]) - 1; kolom >= 0; kolom--)
			{
				printf("%c", string[baris][kolom]);
			}

			printf("\n");
		}

		else{
			
			printf("%s\n", string[baris]);
		}
		
	}

	return 0;
}